### ISSUE TITLE
Ursula Status Page 500

### ISSUE THREAD

**Issue by KPrasch** [nucypher/nucypher#1603]
```
Internal Server Error
The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.
```

```
	Traceback (most recent call last):
	  File "/home/ubuntu/nucypher/nucypher/network/server.py", line 376, in status
	    checksum_address=this_node.checksum_address)
	  File "/home/ubuntu/venv/lib/python3.6/site-packages/jinja2/asyncsupport.py", line 76, in render
	    return original_render(self, *args, **kwargs)
	  File "/home/ubuntu/venv/lib/python3.6/site-packages/jinja2/environment.py", line 1008, in render
	    return self.environment.handle_exception(exc_info, True)
	  File "/home/ubuntu/venv/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
	    reraise(exc_type, exc_value, tb)
	  File "/home/ubuntu/venv/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
	    raise value.with_traceback(tb)
	  File "<template>", line 147, in top-level template code
	jinja2.exceptions.UndefinedError: 'nucypher.network.nodes.NodeSprout object' has no attribute 'rest_url'
	
	The above exception was the direct cause of the following exception:
	
	Traceback (most recent call last):
	  File "/home/ubuntu/venv/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
	    response = self.full_dispatch_request()
	  File "/home/ubuntu/venv/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
	    rv = self.handle_user_exception(e)
	  File "/home/ubuntu/venv/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
	    reraise(exc_type, exc_value, tb)
	  File "/home/ubuntu/venv/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
	    raise value
	  File "/home/ubuntu/venv/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
	    rv = self.dispatch_request()
	  File "/home/ubuntu/venv/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
	    return self.view_functions[rule.endpoint](**req.view_args)
	  File "/home/ubuntu/nucypher/nucypher/network/server.py", line 379, in status
	    raise TemplateError(str(e)) from e
	jinja2.exceptions.TemplateError: 'nucypher.network.nodes.NodeSprout object' has no attribute 'rest_url'

```

**Comment by afmsavage** [comment]
Not sure if this helps, but when I am getting this as well.  When I run in interactive mode and try to do `known_nodes`, I get the following errors:

```Known Nodes (connected 221 / seen 152)
Fleet State 896296b ⇀RoyalBlue Helm↽ ⎈
Unhandled Error
Traceback (most recent call last):
  File "/home/tony/nu/nucypher-venv/lib/python3.6/site-packages/twisted/python/log.py", line 103, in callWithLogger
    return callWithContext({"system": lp}, func, *args, **kw)
  File "/home/tony/nu/nucypher-venv/lib/python3.6/site-packages/twisted/python/log.py", line 86, in callWithContext
    return context.call({ILogContext: newCtx}, func, *args, **kw)
  File "/home/tony/nu/nucypher-venv/lib/python3.6/site-packages/twisted/python/context.py", line 122, in callWithContext
    return self.currentContext().callWithContext(ctx, func, *args, **kw)
  File "/home/tony/nu/nucypher-venv/lib/python3.6/site-packages/twisted/python/context.py", line 85, in callWithContext
    return func(*args,**kw)
--- <exception caught here> ---
  File "/home/tony/nu/nucypher-venv/lib/python3.6/site-packages/twisted/internet/posixbase.py", line 614, in _doReadOrWrite
    why = selectable.doRead()
  File "/home/tony/nu/nucypher-venv/lib/python3.6/site-packages/twisted/internet/process.py", line 252, in doRead
    return fdesc.readFromFD(self.fd, self.dataReceived)
  File "/home/tony/nu/nucypher-venv/lib/python3.6/site-packages/twisted/internet/fdesc.py", line 94, in readFromFD
    callback(output)
  File "/home/tony/nu/nucypher-venv/lib/python3.6/site-packages/twisted/internet/process.py", line 256, in dataReceived
    self.proc.childDataReceived(self.name, data)
  File "/home/tony/nu/nucypher-venv/lib/python3.6/site-packages/twisted/internet/_posixstdio.py", line 77, in childDataReceived
    self.protocol.dataReceived(data)
  File "/home/tony/nu/nucypher-venv/lib/python3.6/site-packages/twisted/protocols/basic.py", line 572, in dataReceived
    why = self.lineReceived(line)
  File "/home/tony/nu/nucypher-venv/lib/python3.6/site-packages/nucypher/cli/processes.py", line 148, in lineReceived
    self.__commands[line]()
  File "/home/tony/nu/nucypher-venv/lib/python3.6/site-packages/nucypher/cli/processes.py", line 96, in paintKnownNodes
    paint_known_nodes(emitter=self.emitter, ursula=self.ursula)
  File "/home/tony/nu/nucypher-venv/lib/python3.6/site-packages/nucypher/cli/painting.py", line 182, in paint_known_nodes
    emitter.echo(row_template.format(node.rest_url().ljust(20), node), color=color_index[node_type])
builtins.AttributeError: 'NodeSprout' object has no attribute 'rest_url'
```

Here's my status:

```Ursula(0xe481553) >>> status

⇀URSULA ⏚ ☕↽
(Ursula)⇀SeaGreen Ground DeepSkyBlue Beverage↽ (0xe48155371907D7dC7E82816446e55c49536333DF)
Uptime .............. 0:02:26
Start Time .......... 2 minutes ago
Fleet State.......... 896296b ⇀RoyalBlue Helm↽ ⎈
Learning Status ..... Learning at 5s Intervals
Learning Round ...... Round #31
Operating Mode ...... Decentralized
Rest Interface ...... 138.197.79.41:9151
Node Storage Type ... Local
Known Nodes ......... 221
Work Orders ......... 0
Current Teacher ..... (NodeSprout)⇀Khaki Ferry Black Star↽ (0x1A63161758Bed2e5903752169F81DFC48e259F54)
Current Period ...... 18283
Worker Address ...... 0x6ea4767a77363445e93De85401bd8BC033182bb8```

**Comment by jMyles** [comment]
So, there are really three things here:

1) this code path is untested
2) we *probably* want to `mature()` this node
3) we can add the attribute access feature in BSS

Somebody want to direct me which of these trees up which they want me to bark?

**Comment by KPrasch** [comment]
Opened a draft  - <https://github.com/nucypher/nucypher/pull/1615>. Feel free to use it as a base for more.

**Comment by KPrasch** [comment]
Fixed, but still untested in #1615
